server/app/controllers/migration.controller.js

```diff
@@ -356,7 +356,7 @@ const updateStatus = async (req, res) => {
                 }});
             }
         } else {
-            res.status(404).json({ error: "Conversion Status Profile you are trying to update does not exists" });
+            res.status(200).json({ message: "Profile you are trying to update does not exists." });
```

Comment:
Should it not be 500 for that

Reply:
500 we are only sending when there is some error, that is mostly the runtime error, for this case the query ran successfully on DB but the data on the query is not available. So we are updating Client accordingly.

Reply:
Yes as we discussed about status and UI code, update it with 500 for that

Reply:
updated